### PR TITLE
Elite Count Can Probably Be Saved

### PR DESCRIPTION
**Pull request by thebeanogamer**
This won't work yet

`ctx.guild.get_role(` needs to be added, and I'm undecided as to where. Also that line in constants is now far too long.

### CODE REVIEW

**Comment by thebeanogamer** [comment]
This _should_ now work, I'll test it properly in a moment.

**Comment by thebeanogamer** [comment]
image

🙌 It works, now to make it compliant.

File: cdbot/constants.py [hunk 1]
```diff
@@ -43,15 +43,30 @@ class PostgreSQL:
...
-ELITE_BRM_OLD_ROLE_ID = 580387899833581572
-ELITE_LAN_OLD_ROLE_ID = 580387898973618176
-ELITE_LDN_YNG_ROLE_ID = 580387877385404428
-ELITE_BRM_YNG_ROLE_ID = 580387895299276830
-ELITE_LAN_YNG_ROLE_ID = 580387892853997578
-ELITE_EXCH_LIST_ROLE_ID = 582894164597932034
-ELITE_EXCH_CONF_ROLE_ID = 585150522336608256
+
+
+class ROLES:
```
Comment (AnonGuy) [inline cdbot/constants.py]: https://www.python.org/dev/peps/pep-0008/#class-names

File: cdbot/constants.py [hunk 2]
```diff
@@ -43,15 +43,30 @@ class PostgreSQL:
...
+
+    class ELITE:
```
Comment (AnonGuy) [inline cdbot/constants.py]: https://www.python.org/dev/peps/pep-0008/#class-names

File: cdbot/constants.py [hunk 3]
```diff
@@ -43,15 +43,30 @@ class PostgreSQL:
...
+
+        ID = int(environ.get("ELITE_MEMBERS_ID", "580387468336037888"))
+
+        class LONDON:
```
Comment (AnonGuy) [inline cdbot/constants.py]: https://www.python.org/dev/peps/pep-0008/#class-names

File: cdbot/constants.py [hunk 4]
```diff
@@ -43,15 +43,30 @@ class PostgreSQL:
...
+            YOUNGER = int(environ.get("LDN_Y_MEMBERS_ID", "580387877385404428"))
+            OLDER = int(environ.get("LDN_O_MEMBERS_ID", "580387897644023811"))
+
+        class BIRMINGHAM:
```
Comment (AnonGuy) [inline cdbot/constants.py]: https://www.python.org/dev/peps/pep-0008/#class-names

File: cdbot/constants.py [hunk 5]
```diff
@@ -43,15 +43,30 @@ class PostgreSQL:
...
+            YOUNGER = int(environ.get("BRM_Y_MEMBERS_ID", "580387895299276830"))
+            OLDER = int(environ.get("BRM_O_MEMBERS_ID", "580387899833581572"))
+
+        class LANCASTER:
```
Comment (AnonGuy) [inline cdbot/constants.py]: https://www.python.org/dev/peps/pep-0008/#class-names

File: cdbot/constants.py [hunk 6]
```diff
@@ -43,15 +43,30 @@ class PostgreSQL:
...
+            YOUNGER = int(environ.get("LAN_Y_MEMBERS_ID", "580387898973618176"))
+            OLDER = int(environ.get("LAN_O_MEMBERS_ID", "580387892853997578"))
+
+    class EXCHANGE:
```
Comment (AnonGuy) [inline cdbot/constants.py]: https://www.python.org/dev/peps/pep-0008/#class-names

File: cdbot/constants.py [hunk 7]
```diff
@@ -43,15 +43,30 @@ class PostgreSQL:
...
-ELITE_BRM_YNG_ROLE_ID = 580387895299276830
-ELITE_LAN_YNG_ROLE_ID = 580387892853997578
-ELITE_EXCH_LIST_ROLE_ID = 582894164597932034
-ELITE_EXCH_CONF_ROLE_ID = 585150522336608256
+
+
+class ROLES:
+
+    class ELITE:
+
```
Comment (AnonGuy) [inline cdbot/constants.py]: Pls remove this newline

Comment (AnonGuy) [reply cdbot/constants.py]: (Line 51, to be precise)

**Comment by AnonGuy** [comment]
See Joseph's PostgreSQL class to see how enums and their constants should be formatted:
```python
class PostgreSQL:
    PGHOST = getenv("PGHOST")
    PGPORT = getenv("PGPORT")
    PGUSER = getenv("PGUSER")
    PGDATABASE = getenv("PGDATABASE")
    PGPASSWORD = getenv("PGPASSWORD")
```

**Comment by AnonGuy** [comment]
wut

File: cdbot/constants.py [hunk 8]
```diff
@@ -43,15 +43,29 @@ class PostgreSQL:
...
-ELITE_BRM_YNG_ROLE_ID = 580387895299276830
-ELITE_LAN_YNG_ROLE_ID = 580387892853997578
-ELITE_EXCH_LIST_ROLE_ID = 582894164597932034
-ELITE_EXCH_CONF_ROLE_ID = 585150522336608256
+
+
+class Roles:
+
+    class Elite:
+        Main = int(environ.get("ELITE_MEMBERS_ID", "580387468336037888"))
```
Comment (AnonGuy) [inline cdbot/constants.py]: These attributes should be all caps:
https://www.python.org/dev/peps/pep-0008/#constants

File: cdbot/constants.py [hunk 9]
```diff
@@ -43,15 +43,29 @@ class PostgreSQL:
...
+
+        class London:
+            Younger = int(environ.get("LDN_Y_MEMBERS_ID", "580387877385404428"))
```
Comment (AnonGuy) [inline cdbot/constants.py]: These attributes should be all caps:
https://www.python.org/dev/peps/pep-0008/#constants

File: cdbot/constants.py [hunk 10]
```diff
@@ -43,15 +43,29 @@ class PostgreSQL:
...
+            Younger = int(environ.get("BRM_Y_MEMBERS_ID", "580387895299276830"))
+            Older = int(environ.get("BRM_O_MEMBERS_ID", "580387899833581572"))
+
+        class Lancaster:
+            Younger = int(environ.get("LAN_Y_MEMBERS_ID", "580387898973618176"))
+            Older = int(environ.get("LAN_O_MEMBERS_ID", "580387892853997578"))
+
+    class Exchange:
+        Shortlist = int(environ.get("EXCH_S_MEMBERS_ID", "582894164597932034"))
+        Confirmed = int(environ.get("EXCH_C_MEMBERS_ID", "585150522336608256"))
```
Comment (AnonGuy) [inline cdbot/constants.py]: These attributes should be all caps:
https://www.python.org/dev/peps/pep-0008/#constants

File: cdbot/constants.py [hunk 11]
```diff
@@ -43,15 +43,29 @@ class PostgreSQL:
...
+        class Birmingham:
+            Younger = int(environ.get("BRM_Y_MEMBERS_ID", "580387895299276830"))
+            Older = int(environ.get("BRM_O_MEMBERS_ID", "580387899833581572"))
+
+        class Lancaster:
+            Younger = int(environ.get("LAN_Y_MEMBERS_ID", "580387898973618176"))
+            Older = int(environ.get("LAN_O_MEMBERS_ID", "580387892853997578"))
+
+    class Exchange:
+        Shortlist = int(environ.get("EXCH_S_MEMBERS_ID", "582894164597932034"))
```
Comment (AnonGuy) [inline cdbot/constants.py]: These attributes should be all caps:
https://www.python.org/dev/peps/pep-0008/#constants

File: cdbot/constants.py [hunk 12]
```diff
@@ -43,15 +43,29 @@ class PostgreSQL:
...
+            Younger = int(environ.get("LDN_Y_MEMBERS_ID", "580387877385404428"))
+            Older = int(environ.get("LDN_O_MEMBERS_ID", "580387897644023811"))
+
+        class Birmingham:
+            Younger = int(environ.get("BRM_Y_MEMBERS_ID", "580387895299276830"))
+            Older = int(environ.get("BRM_O_MEMBERS_ID", "580387899833581572"))
+
+        class Lancaster:
+            Younger = int(environ.get("LAN_Y_MEMBERS_ID", "580387898973618176"))
+            Older = int(environ.get("LAN_O_MEMBERS_ID", "580387892853997578"))
```
Comment (AnonGuy) [inline cdbot/constants.py]: These attributes should be all caps:
https://www.python.org/dev/peps/pep-0008/#constants

File: cdbot/constants.py [hunk 13]
```diff
@@ -43,15 +43,29 @@ class PostgreSQL:
...
+        class London:
+            Younger = int(environ.get("LDN_Y_MEMBERS_ID", "580387877385404428"))
+            Older = int(environ.get("LDN_O_MEMBERS_ID", "580387897644023811"))
+
+        class Birmingham:
+            Younger = int(environ.get("BRM_Y_MEMBERS_ID", "580387895299276830"))
+            Older = int(environ.get("BRM_O_MEMBERS_ID", "580387899833581572"))
+
+        class Lancaster:
+            Younger = int(environ.get("LAN_Y_MEMBERS_ID", "580387898973618176"))
```
Comment (AnonGuy) [inline cdbot/constants.py]: These attributes should be all caps:
https://www.python.org/dev/peps/pep-0008/#constants

File: cdbot/constants.py [hunk 14]
```diff
@@ -43,15 +43,29 @@ class PostgreSQL:
...
+    class Elite:
+        Main = int(environ.get("ELITE_MEMBERS_ID", "580387468336037888"))
+
+        class London:
+            Younger = int(environ.get("LDN_Y_MEMBERS_ID", "580387877385404428"))
+            Older = int(environ.get("LDN_O_MEMBERS_ID", "580387897644023811"))
+
+        class Birmingham:
+            Younger = int(environ.get("BRM_Y_MEMBERS_ID", "580387895299276830"))
+            Older = int(environ.get("BRM_O_MEMBERS_ID", "580387899833581572"))
```
Comment (AnonGuy) [inline cdbot/constants.py]: These attributes should be all caps:
https://www.python.org/dev/peps/pep-0008/#constants

File: cdbot/constants.py [hunk 15]
```diff
@@ -43,15 +43,29 @@ class PostgreSQL:
...
+
+    class Elite:
+        Main = int(environ.get("ELITE_MEMBERS_ID", "580387468336037888"))
+
+        class London:
+            Younger = int(environ.get("LDN_Y_MEMBERS_ID", "580387877385404428"))
+            Older = int(environ.get("LDN_O_MEMBERS_ID", "580387897644023811"))
+
+        class Birmingham:
+            Younger = int(environ.get("BRM_Y_MEMBERS_ID", "580387895299276830"))
```
Comment (AnonGuy) [inline cdbot/constants.py]: These attributes should be all caps:
https://www.python.org/dev/peps/pep-0008/#constants

File: cdbot/constants.py [hunk 16]
```diff
@@ -43,15 +43,29 @@ class PostgreSQL:
...
+
+
+class Roles:
+
+    class Elite:
+        Main = int(environ.get("ELITE_MEMBERS_ID", "580387468336037888"))
+
+        class London:
+            Younger = int(environ.get("LDN_Y_MEMBERS_ID", "580387877385404428"))
+            Older = int(environ.get("LDN_O_MEMBERS_ID", "580387897644023811"))
```
Comment (AnonGuy) [inline cdbot/constants.py]: These attributes should be all caps:
https://www.python.org/dev/peps/pep-0008/#constants

**Pull request merged**